package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nuxt-nitro-storage",
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
the versioning is automatically done with npm so no need to change it manually

Reply:
Didn't know that